### PR TITLE
Ubuntu Touch: adapt to 20.04

### PR DESCRIPTION
**Pull request by lduboeuf**

### CODE REVIEW

File: platforms/ubuntutouch/clickable.yml [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+  - /usr/lib/${ARCH_TRIPLET}/qt5/qml/Qt/labs/calendar
+make_jobs: 4
+libraries:
+  Box2D:
+    builder: qmake
+    make_jobs: 2
+    src_dir: external/qml-box2d
+    prebuild: git submodule update --init --recursive
+
+  QtQuickCalendar:
```
Comment (petitlapin) [inline platforms/ubuntutouch/clickable.yml:18]: this one does not need to be built, we ae not using it when building with Qt5

File: platforms/ubuntutouch/gcompris.desktop [hunk 2]
```diff
@@ -88,7 +88,7 @@ GenericName[ta]=குழந்தைகளுக்கான கல்விச
 GenericName[tg]=Бозиҳои омӯзишӣ барои кӯдакон
 GenericName[tr]=Çocuklar için Eğitici Oyunlar
 GenericName[uk]=Освітня гра для дітей
-GenericName[x-test]=xxEducational Game for Childrenxx
+#GenericName[x-test]=xxEducational Game for Childrenxx
```
Comment (petitlapin) [inline platforms/ubuntutouch/gcompris.desktop:91]: we won't be able to commit this change as this part of the file is filled automatically by KDE scripts and we can't ignore it.

File: platforms/ubuntutouch/clickable.yml [hunk 3]
```diff
@@ -0,0 +1,24 @@
+clickable_minimum_required: "7.6"
+builder: cmake
+kill: gcompris-qt
+build_args: "-DUBUNTU_TOUCH=ON"
+dependencies_target:
+  - libqt5charts5-dev
```
Comment (petitlapin) [inline platforms/ubuntutouch/clickable.yml:6]: I think you also need to install `qml-module-charts` else fractions activities won't work

**Review by petitlapin — CHANGES_REQUESTED** [review]
Thank you for the PR.

I've made some comments. When it's good, I'll report them to the KDE source code repository.

File: platforms/ubuntutouch/gcompris.desktop (hunk 2 above)
Comment (lduboeuf) [reply platforms/ubuntutouch/gcompris.desktop:91]: that desktop file is a copy of the root dir one. Is it still an issue ?. 
I can rollback it if needed. This is just that our build system doesn't support x-test name for whatever reason

File: platforms/ubuntutouch/clickable.yml (hunk 3 above)
Comment (lduboeuf) [reply platforms/ubuntutouch/clickable.yml:6]: this is also something missing in the build cli of ubuntu touch. i need to add the dev package whereas the module is already installed on devices

File: platforms/ubuntutouch/gcompris.desktop (hunk 2 above)
Comment (lduboeuf) [reply platforms/ubuntutouch/gcompris.desktop:91]: yes it fails when building.

Comment (petitlapin) [reply platforms/ubuntutouch/gcompris.desktop:91]: According to the KDE i18n team:
```
We don't
but we should really remove the x-test translations at this point
there's just no way to use them at this point
```

I'll take a look if I can help them to remove these ones

File: platforms/ubuntutouch/manifest.json.in [hunk 4]
```diff
@@ -11,5 +11,5 @@
     },
     "version": "@GCOMPRIS_MAJOR_VERSION@.@GCOMPRIS_MINOR_VERSION@",
     "maintainer": "lionel <lduboeuf@ouvaton.org>",
-    "framework" : "ubuntu-sdk-16.04"
+    "framework" : "@CLICK_FRAMEWORK@"
```
Comment (petitlapin) [inline platforms/ubuntutouch/manifest.json.in:14]: where is this variable filled?

Comment (petitlapin) [reply platforms/ubuntutouch/manifest.json.in:14]: ok I have it https://clickable-ut.dev/en/stable/usage.html#app-manifest

**Comment by petitlapin** [comment]
Thank you!
I've reported the commit in our main repository: https://invent.kde.org/education/gcompris/-/commit/50cc9dbe710f72e473cfa2fc14a4d743391a1f17
Except for the gcompris.desktop that will be reverted automatically and I need to find a solution to remove x-test.

**Pull request closed without merging**